private/model/api/codegentest/service/pagin8/paginator_test.go

```diff
@@ -0,0 +1,639 @@
+package pagin8
```

Comment:
Nit what do you think bout naming this file something like `cust_paginator_test.go` since its hand written and the other tests under codegentest are codegen.